packages/node_modules/samples/browser-meetings/index.html

```diff
@@ -0,0 +1,367 @@
+        display: flex;
+        justify-content: center;
+        flex-wrap: wrap;
+      }
+
+      video {
+        height: 20vh;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>THIS IS A WORK IN PROGRESS SAMPLE</h1>
+    <section>
+      <h2>Meetings</h2>
+      <p>[[APPLICATION DESCRIPTION]]</p>
+    </section>
+
+    <section>
+      <h2>Authentication</h2>
+      <form id="credentials">
+        <fieldset>
+          <legend>Credentials</legend>
+          <input id="access-token" name="accessToken" placeholder="Access Token" value="" type="text" />
+          <input id="access-token-save" title="save" type="submit" value="Save" />
+          <label id="access-token-status">[[CREDENTIALS STATUS]]</label>
+        </fieldset>
+      </form>
+
+      <br />
+
+      <form>
+        <fieldset>
+          <legend>Registration</legend>
+          <input id="registration-register" title="register" type="submit" value="Register" />
+          <input id="registration-unregister" title="unregister" type="submit" value="Unregister" />
+          <label id="registration-status">[[REGISTRATION STATUS]]</label>
+        </fieldset>
+      </form>
+    </section>
+
+    <section>
+      <h2>Meetings Management</h2>
+      <form id="meetings-create">
+        <fieldset>
+          <legend>Create a Meeting</legend>
+          <input id="create-meeting-destination" name="createMeetingDestination" placeholder="Destination" value="" type="text" />
+          <input id="create-meeting-action" title="create" type="submit" value="Create" />
+        </fieldset>
+      </form>
+
+      <br />
+
+      <form id="meetings-list">
+        <fieldset>
+          <legend>Current Meetings List</legend>
+          <div>
+            <input id="meetings-join-a-wom" title="joinWOM" type="submit" value="Join Without Media" />
+            <input id="meetings-join-a-wm" title="joinWM" type="submit" value="Join With Media" />
+            <label>[[MEETING IDENTIFIER]]</label>
+          </div>
+          <div>
+            <input id="meetings-join-b-wom" title="joinWOM" type="submit" value="Join Without Media" />
+            <input id="meetings-join-b-wm" title="joinWM" type="submit" value="Join With Media" />
+            <label>[[MEETING IDENTIFIER]]</label>
+          </div>
+          <div>
+            <input id="meetings-join-c-wom" title="joinWOM" type="submit" value="Join Without Media" />
+            <input id="meetings-join-c-wm" title="joinWM" type="submit" value="Join With Media" />
+            <label>[[MEETING IDENTIFIER]]</label>
+          </div>
+        </fieldset>
+      </form>
+
+      <br />
+
+      <form id="current-meeting">
+        <fieldset>
+          <legend>Current Joined Meeting</legend>
+          <input id="meetings-leave" title="leave" type="submit" value="Leave" />
+          <label>[[CURRENT MEETING IDENTIFIER]]</label>
+        </fieldset>
+      </form>
+    </section>
+
+    <section>
+      <h2>Meetings Devices</h2>
```

Comment:
same as above comment

```suggestion
      <h2>Meeting Devices</h2>
      <blockquote>[[MEETING Devices DESCRIPTION]]</blockquote>
```